dashboard/serializers.py

```diff
@@ -47,5 +53,6 @@ def serialize(cls, program_enrollment):
             'id': program.id,
             'enrollments': program_cached_enrollments,
             'certificates': program_cached_certificates,
-            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates)
+            'grade_average': cls.calculate_certificate_grade_average(program_cached_certificates),
+            'is_staff': has_role(user, [Staff.ROLE_ID, Instructor.ROLE_ID])
```

Comment:
I think that you should find a way to define the roles that are not `learners` directly in the `role.models` and just list them here.
The reason is because we do not want forget to add here a new role if we create one.